app/models/note.server.ts

```diff
@@ -1,22 +1,28 @@
-import cuid from "cuid";
 import arc from "@architect/functions";
+import cuid from "cuid";
+import { User } from "./user.server";
```

Comment:
Since we're only using the type, let's import just the type

```suggestion

import type { User } from "./user.server";
```